geode-apis-compatible-with-redis/src/test/java/org/apache/geode/redis/internal/data/RedisHashTest.java

```diff
@@ -494,42 +452,9 @@ public void sizeShouldDecrease_whenValueIsRemoved() {
 
     redisHash.hdel(region, key, dataToRemove);
 
-    int expectedSize = initialSize - RedisHash.HASH_MAP_VALUE_PAIR_OVERHEAD - field1.length;
-    Offset<Integer> offset = Offset.offset((int) round(expectedSize * 0.05));
+    long expectedSize = initialSize - (elementSizer.sizeof(field1) + elementSizer.sizeof(value1));
```

Comment:
I feel similarly about this one...  It just seems like this exactly mimics our underlying implementation...which seems bad.  Could we get the initial size and assert that it's less than the final size... then maybe check that the elementSizer size of `redisHash` after `hdel` is equal to the `getSizeInBytes` of `redisHash`?

Reply:
Agreed, I tweaked the assertions.